src/uproot/sink/file.py

```diff
@@ -9,11 +9,26 @@
 (although files are flushed after every object-write).
 """
 
+from __future__ import annotations
 
 import numbers
 import os
 
 
+def _is_file_like(obj) -> bool:
+    return (
+        callable(getattr(obj, "read", None))
+        and callable(getattr(obj, "write", None))
+        and callable(getattr(obj, "seek", None))
+        and callable(getattr(obj, "tell", None))
+        and callable(getattr(obj, "flush", None))
+        # TODO: what was the motivation behind the checks below? (need to remove them for some fsspec backends)
+        # and (not hasattr(obj, "readable") or obj.readable())
+        # and (not hasattr(obj, "writable") or obj.writable())
+        # and (not hasattr(obj, "seekable") or obj.seekable())
```

Comment:
@jpivarski do you recall why these lines were added? Is it okay to remove them? Some of the fsspec file-like objects (such as the ssh one) implement these interfaces so this wouldn't recognize it as a file-like object.